packages/@lightningjs/ui-components/src/components/MetadataTile/MetadataTile.test.js

```diff
@@ -67,11 +67,15 @@ describe('MetadataTile', () => {
 
   it('hides details if description is provided', async () => {
     metadataTile.details = 'details';
-    await metadataTile.__updateSpyPromise;
+    await metadataTile.__detailsLoadedpyPromise;
```

Comment:
I think there is an S missing in the `_detailsLoadedpyPromise`

Reply:
womp. Thanks!